doc/content/getting-started/migrating-from-v2/configure-mac-settings.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Configure End Devices
+weight: 60
+---
+
+{{< cli-only >}}
+
+This section is optional.
+
+MAC Parameters on {{% tts %}} are configurable per end device. After you have imported your end devices, you can use the CLI to configure their MAC settings.
+
+```bash
+$ ttn-lw-cli end-devices update --help
+```
+
+This will print a list of all available end device options, including MAC settings.
+
+For example, to change the `Rx1Delay` setting for an end device, you can do:
+
+```bash
+$ ttn-lw-cli end-devices update "AppID" "DeviceID" --mac-settings.rx1-delay RX_DELAY_5
+```
+
+You can also unset settings:
+
+```bash
+$ ttn-lw-cli end-devices update "AppID" "DeviceID" --unset mac-settings.rx1-delay
+```
+
+This way you will be able to fine tune your devices so that they work as expected with the Network Server of {{% tts %}}.
```

Comment:
I understand the goal of this guide. I would suggest changing it from something generic about MAC state, to a more optional but recommended guide about finetuning MAC state for V3.

That would definitely include changing the RX1 delay to 5 seconds.

@rvolosatovs is there any other desired MAC parameter that we want to mention here?

Reply:
Nothing specific to migration, I think we should just have a generic list of v3 MAC settings with use cases and reference it here.

Reply:
Will create an issue and add a TODO comment for this.